tools/toolchains/iar.py

```diff
@@ -140,17 +140,26 @@ def get_config_option(self, config_header):
 
     def get_compile_options(self, defines, includes, for_asm=False):
         opts = ['-D%s' % d for d in defines]
-        if for_asm :
+        if for_asm:
```

Comment:
@TTornblom Can you review this change in this file , regarding `-f` for assembly? There's related issue https://github.com/ARMmbed/mbed-os/issues/4879 that we faced again in the recent days.